src/codegen/object.rs

```diff
@@ -287,7 +287,7 @@ impl {}Builder {{
         writeln!(
             w,
             r#"        let ret = {}::Object::new::<{}>(&properties)
-                .expect("Failed to create an instance of {}"");"#,
+                .expect("Failed to create an instance of {}");"#,
```

Comment:
The commit message says to add a missing `}`, the commit removes an additional `"` :)

Reply:
there are two commits :-)

Reply:
sorry, i completely messed with the latest PR